packaging/README-dockerhub.md

```diff
@@ -39,3 +41,8 @@ different commit or tag, specify the `SOURCE_COMMIT` build argument:
 ```shell
 sudo docker build --no-cache --build-arg SOURCE_COMMIT=v0.0.4 -t nmstate/centos7-nmstate -f Dockerfile.centos7-nmstate .
 ```
+
+```shell
+./build-container.sh --extra-args "--build-arg SOURCE_COMMIT=v0.0.6" local/centos7-nmstate
```

Comment:
Could you please clarify how is this change related? What is local/centos7-nmstate used for?

Reply:
The `local` prefix comes from you. I moved the build instructions from the README here:
https://github.com/nmstate/nmstate/commit/a2011b00326c0989834559a7d624061c081068f2